src/utils.domNodeDisposal.js

```diff
@@ -85,7 +90,7 @@ ko.utils.domNodeDisposal = new (function () {
                 node.parentNode.removeChild(node);
         },
 
-        "cleanExternalData" : function (node) {
+        cleanExternalData : function (node) {
```

Comment:
I just found it weird that this name was quoted when none of the others were. Should it be in the ko.exportSymbol list at the bottom of the file?